### PR TITLE
feature - current address

### PR DESCRIPTION
**Pull request by joefitter**
* added hof-behaviour-address-lookup
* added acceptance test
* added Dockerfile-acceptance

### CODE REVIEW

File: apps/right-to-rent-check/index.js [hunk 1]
```diff
@@ -17,12 +19,12 @@ module.exports = {
...
+      next: '/current-property-address'
     },
     '/current-property-address': {
-      next: '/current-property-address-select'
-    },
-    '/current-property-address-select': {
+      behaviours: AddressLookup({
+        addressKey: 'current-address'
+      }),
       next: '/tenant-details'
```
Comment (sulthan-ahmed) [inline apps/right-to-rent-check/index.js]: this is crazy simple 👍

Comment (joefitter) [inline apps/right-to-rent-check/index.js]: I NO RITE

File: package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "watch:scss": "nodemon -e scss -x 'npm run build:sass'",
     "watch:translations": "nodemon --watch apps/**/translations/src -x 'npm run build:translate'",
     "build:translate": "hof-transpiler ./apps/**/translations/src -w",
-    "test": "npm-run-all test:lint test:acceptance",
+    "test": "npm-run-all test:lint",
```
Comment (sulthan-ahmed) [inline package.json]: guess you don't need the npm-run-all if you're running one command

Comment (joefitter) [inline package.json]: good spot

**Pull request merged**